exporter/elasticsearchexporter/logs_exporter.go

```diff
@@ -18,9 +19,12 @@ import (
 type elasticsearchLogsExporter struct {
 	logger *zap.Logger
 
-	index        string
-	dynamicIndex bool
-	maxAttempts  int
+	// if logstash format enabled, this index will ignore and won't be used.
```

Comment:
Can this info also be displayed in README.md?

Reply:
Since only one option between `logstash` and `index` can be respected, should we validate this in config validate function?

Reply:
thx, this info outdated, so, I removed it.